safe-global/safe-wallet-web
# fix: Add social login feature toggle from config service

## What it solves

Resolves #2458 

## How this PR fixes it

- Only shows social login in the onboard modal if its enabled in the config service
- The info message now displays all supported chains depending on the config service
- If the user is connected with a different wallet, the social login button will no longer show a connected state 

## How to test it

1. Open the welcome page on goerli staging CGW
2. Connect via Google
3. Connect to a different wallet either through the main button or the "Switch Wallet" button
4. Observe the Google button on the welcome page says "Continue with Google"
5. Switch to Polygon
6. Observe an info message saying that social login is only supported on Goerli
7. Open the Onboard modal
8. Observe no Social Login option

## Screenshots
<img width="446" alt="Screenshot 2023-10-16 at 11 55 29" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/6ab3edba-6299-4970-8f69-84829ca57818">
<img width="590" alt="Screenshot 2023-10-16 at 11 56 45" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/f31fa242-1858-42c0-8332-f4587e371748">
<img width="1161" alt="Screenshot 2023-10-16 at 11 57 06" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/e7ed1646-e361-4858-939e-78b20ba29e25">

## Checklist
